rest/server_context.go

```diff
@@ -1207,9 +1207,9 @@ var tempConnectionDetailsForManagementEndpoints = func() (serverAddress string,
 	return base.UnitTestUrl(), base.TestClusterUsername(), base.TestClusterPassword(), "", "", ""
 }
 
-func (sc *ServerContext) ObtainManagementEndpointsAndHTTPClient() ([]string, *http.Client, error) {
+func (sc *ServerContext) ObtainManagementEndpointsAndHTTPClient(CACertUnsetTlsSkipVerify bool) ([]string, *http.Client, error) {
```

Comment:
I think we'd pull tlsSkipVerify from the same place we get the rest of the tlsConfig (currently tempConnectionDetails), and not pass into this method.

Reply:
Agreed that aligns with it being a server config item.